var/spack/repos/builtin/packages/rocm-tensile/002-fix-boost-inc-dirs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Tensile/Source/client/CMakeLists.txt b/Tensile/Source/client/CMakeLists.txt
+index 9ccba86..59ed2e2 100644
+--- a/Tensile/Source/client/CMakeLists.txt
++++ b/Tensile/Source/client/CMakeLists.txt
+@@ -32,7 +32,7 @@ if(NOT ROCmSMI_FOUND)
+     find_package(ROCmSMI REQUIRED)
+ endif()
+ 
+-target_include_directories(TensileClient PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/include")# "${rocm_smi_root}/include")
++target_include_directories(TensileClient PUBLIC "${Boost_INCLUDE_DIRS}" "${CMAKE_CURRENT_SOURCE_DIR}/include")# "${rocm_smi_root}/include")
```

Comment:
This is not necessary, it uses `find_package(Boost  COMPONENTS program_options filesystem REQUIRED)`, should not require a patch.

Reply:
Thanks for the review. Build fails without adding this patch. Is there a way around this?